### ISSUE TITLE
ML - Training Script - Logistic Regression

### ISSUE THREAD

**Issue by Mohammad4844** [COSC481W-2024Fall/MLite#7]
## User Story
As a user, I would like to train a logistic regression model. I want to be able to go to my dataset, select a categorical label, and train a Logistic Regression model.

### Persona 
Existing users looking to train a model.

### Feature
Be able to run a training job through our app, specifically for a logistic regression model.

### Business Value
Users can train a new model type from our app for datasets with categorical labels.

## Tasks

1. Script takes in input hyperparameters
2. Script loads the dataset
3. Script initializes a logistic regression model & trains on the dataset
4. Script saves/uploads the trained model

## Acceptance Criteria
- Will NOT implement integration with our Rails app. Simply ML scripts to train a given model.

## Acceptance Tests

- Users can schedule a Logistic Regression model training
- Training runs successfully
- Final model is saved in an appropriate location

**Comment by congbrian** [comment]
Needs review, pushed in 348980c6feab2e60697bb74d3637687e36a5eff5
